src/LocalInstaller.ts

```diff
@@ -1,3 +1,4 @@
+import * as child_process from 'child_process';
```

Comment:
the `ExecOptions` are actually re-exported from `mz/child_process` (using `export * from "child_process";`), so you should be able to use `import { exec, ExecOptions } from 'mz/child_process';` instead.

Reply:
Oww, good to know, much more cleaner that way.